kornia/filters/kernels.py

```diff
@@ -622,21 +622,20 @@ def get_motion_kernel2d(kernel_size: int, angle: Union[torch.Tensor, float],
         torch.Tensor: the motion blur kernel.
 
     Shape:
-        - Output: :math:`(ksize, ksize)`
-
-    Examples:
-        get_motion_kernel2d(5, 0., 0.)
-        tensor([[0.0000, 0.0000, 0.0000, 0.0000, 0.0000],
-                [0.0000, 0.0000, 0.0000, 0.0000, 0.0000],
-                [0.2000, 0.2000, 0.2000, 0.2000, 0.2000],
-                [0.0000, 0.0000, 0.0000, 0.0000, 0.0000],
-                [0.0000, 0.0000, 0.0000, 0.0000, 0.0000]])
-        get_motion_kernel2d(3, 215., -0.5)
-        tensor([[0.0000, 0.0412, 0.0732],
-                [0.1920, 0.3194, 0.0804],
-                [0.2195, 0.0743, 0.0000]])
+        - Output: :math:`(B, ksize, ksize)`
+
+    Examples::
+        >>> get_motion_kernel2d(5, 0., 0.)
+        tensor([[[0.0000, 0.0000, 0.0000, 0.0000, 0.0000],
+                 [0.0000, 0.0000, 0.0000, 0.0000, 0.0000],
+                 [0.2000, 0.2000, 0.2000, 0.2000, 0.2000],
+                 [0.0000, 0.0000, 0.0000, 0.0000, 0.0000],
+                 [0.0000, 0.0000, 0.0000, 0.0000, 0.0000]]])
```

Comment:
@shijianjian what's the lògic for angles here ? I would expect this 90deg. Probably documento need to specify this too.
Is there to compare against ?

Reply:
I did not get it. This kernel is firstly determined by the direction, the rotate by the specific angles as explained in the doc. Do you mean to -90 degrees to let angle ==0 as vertical?

Reply:
yes